misk/src/main/kotlin/misk/web/actions/UpstreamResourceInterceptor.kt

```diff
@@ -72,4 +78,14 @@ class UpstreamResourceInterceptor(
           upstreamBaseUrl.encodedPath().endsWith("/"))
     }
   }
+
+  class Factory @Inject internal constructor() : NetworkInterceptor.Factory {
+    override fun create(action: Action): NetworkInterceptor? {
+      return UpstreamResourceInterceptor(okhttp3.OkHttpClient(), mutableListOf<Mapping>(
+          UpstreamResourceInterceptor.Mapping("/_admin/test/", HttpUrl.parse("http://localhost:8000/")!!),
+          UpstreamResourceInterceptor.Mapping("/_admin/dashboard/", HttpUrl.parse("http://localhost:3100/")!!),
```

Comment:
Since this list is in misk internal code, how do applications add their own dashboards here?

Reply:
This hasn't been built yet but mappings will be dynamically built when Misk evaluates all the injected modules, which ones have web `_admin` components, and then based on likely a simple YAML file adds the mappings. The YAML file will contain keys for the resource location when in development (url with port to the webpack dev server) and in production (pulled from within the Misk JAR file) and which mode it currently is in.